public/js/main.js

```diff
@@ -9,5 +9,10 @@ $(document).ready(function(){
         return false;
       }
     });
+
+    $('#preview-image').change( function(event) {
+        var imgpath = URL.createObjectURL(event.target.files[0]);
```

Comment:
Why need set image default again? 
I'm has required image input not null, if you not choose image on second click, you can see last image after first click 
Explain for me, please !